### PR TITLE
Fixes white space issue when typing values in the grid

### PR DESCRIPTION
**Pull request by dannyparsons**

### CODE REVIEW

**Comment by dannyparsons** [comment]
@maxwellfundi I've changed the code of ucrDataView here. Let me know if it is ok to merge this without conflicting someone.

**Comment by maxwellfundi** [comment]
@dannyparsons  you could merge this, i had not edited the same file as you so there would not have a conflict at all.

File: instat/ucrDataView.vb [hunk 1]
```diff
@@ -400,7 +403,7 @@ Public Class ucrDataView
                 Case "factor"
                     chrCurrentFactorLevels = frmMain.clsRLink.RunInternalScriptGetValue(clsGetFactorLevels.ToScript()).AsCharacter
                     If Not chrCurrentFactorLevels.Contains(strNewValue) Then
-                        MsgBox("Invalid value: " & strNewValue & Environment.NewLine & "This column is: factor. Values must be an existing level of this factor column.", MsgBoxStyle.Exclamation, "Invalid Value")
+                        MsgBox("Invalid value: '" & strNewValue & "'" & Environment.NewLine & "This column is: factor. Values must be an existing level of this factor column.", MsgBoxStyle.Exclamation, "Invalid Value")
```
Comment (maxwellfundi) [inline instat/ucrDataView.vb]: Why was a single  @quotes added here?

Comment (dannyparsons) [reply instat/ucrDataView.vb]: This goes around the invalid text to make it stand out on the message box.

**Pull request merged**